src/WebJobs.Script/Host/ScriptHostManager.cs

```diff
@@ -134,7 +134,7 @@ public virtual ScriptHost Instance
         /// <returns>True if the host can accept invoke requests, false otherwise.</returns>
         public bool CanInvoke()
```

Comment:
>public bool CanInvoke() [](start = 8, length = 23)

Is the goal of this change to "fix" the CanInvoke() so that it doesn't return true until after OnHostInitialize?

Reply:
Yes, primarily. We now have a place to put code that requires the host to be fully initialized/indexed, without forcing that code to wait for all listeners to start up